### DISCUSSION TITLE
How change canvas size???

### DISCUSSION THREAD

**Discussion by dev-SR** [reaviz/reagraph#101] [Q&A]
How can I control the size of the canvas when plotting multiple graphs on the same page? Currently, only the last graph shows and takes up the full screen because there is no option to change the canvas size. 

here is a dummy code:

```tsx
type DomainData = {
	domain: number;
	nodes: Node[];
	edges: Edge[];
};

export default function Home({ domains }: { domains: DomainData[] }) {
	return (
		<div>
			{domains.map((domain) => (
				<div key={domain.domain} >
					<div >
						<h1>Domain {domain.domain}</h1>
					</div>
					<GraphCanvas
						nodes={domain.nodes}
						edges={domain.edges}
						key={domain.domain}
						labelType='all'
						edgeArrowPosition='none'
					/>
				</div>
			))}
		</div>
	);
}
```

**Comment by kaisen-ai** [comment]
Can anyone help with this question? I'm stuck too.

**Reply by popovitsj** [reply]
I think you may have to set a fixed size, such as in this example: https://reagraph.dev/?path=/story/demos-basic--many

**Comment by knowncolor** [comment]
I am also unable to stop the canvas taking the whole page, adding `position: relative` to a containing `div` prevents the graph from showing at all.

Very strange.

**Reply by wilkyrlx** [reply]
using position: fixed worked for me!

**Reply by amcdnl** [reply]
Would you mind PRing this to the docs? I'm sure others would love this info - maybe under a FAQ or Troubleshooting section?
